frontends/ol-components/src/components/Card/Card.tsx

```diff
@@ -103,13 +104,14 @@ const Info = styled.div<{ size?: Size }>`
 `
 
 const titleOpts = {
-  shouldForwardProp: (prop: string) => prop !== "size",
+  shouldForwardProp: (prop: string) => prop !== "lines" && prop !== "size",
 }
-const Title = styled(Linkable, titleOpts)<{ size?: Size }>`
+const Title = styled(Linkable, titleOpts)<{ size?: Size; lines?: number }>`
```

Comment:
**Suggestion:** use the `truncateText` css helper instead of putting an extra div inside Title: 
```tsx
const Title = styled(
  Linkable,
  titleOpts,
)<{ size?: Size; lines?: number }>(({ theme, size, lines = 3 }) => {
  return [
    {
      display: "flex",
      textOverflow: "ellipsis",
      overflow: "hidden",
      margin: 0,
      ...truncateText(lines),
    },
    size === "small"
      ? {
          ...theme.typography.subtitle2,
          height: theme.typography.pxToRem(18 * lines),
        }
      : {
          ...theme.typography.subtitle1,
          height: theme.typography.pxToRem(20 * lines),
        },
  ]
})
```
then just 
```tsx
        <Title
          data-card-link={!!title.href}
          className="MitCard-title"
          size={size}
          {...title}
        />
```
below. (`lines` and `children` will come from `...title`)


**Additionally:** You could do
```
height: `calc(${lines} * ${theme.typography.subtitle1.lineHeight})` 
```
if you want. TS will tell you `lineHeight` might be undefined. We actually do define it for every typography style, and I can see how using it (as here) is useful, so we could make the theme aware of the fact that it is always defined. Happy to help if you wanna do that.

**Aside:** Apparently there are line-height CSS units, `lh` and `rlh` (see https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#numbers_lengths_and_percentages) but their browser support is only ~91% . So sad.